src/NewsService.Mappers/ResponsesMappers/NewsResponseMapper.cs

```diff
@@ -15,33 +15,48 @@ namespace LT.DigitalOffice.NewsService.Mappers.ResponsesMappers
 {
     public class NewsResponseMapper : INewsResponseMapper
     {
-        private IRequestClient<IGetUserDataRequest> _client;
+        private IRequestClient<IGetUserDataRequest> _requestClient;
```

Comment:
today we will talk about where the client's request should be, and finally decided that it is not in the mapper (violation of the principle of single responsibility)